osgp/platform/osgp-adapter-ws-smartmetering/src/main/resources/logback.xml

```diff
@@ -37,4 +37,6 @@
     <!--<appender-ref ref="DB"/>-->
   </root>
 
+  <logger name="org.springframework.ws.server.MessageTracing" level="TRACE" />
```

Comment:
Is dit bewust gedaan?

Reply:
ja die moet nog weg, net als die er boven; dit zorgt ervoor dat de SOAP messages gelogd worden door Spring